keyboards/nopunin10did/jabberwocky/v2/keymaps/via/keymap.c

```diff
@@ -0,0 +1,46 @@
+/* Copyright 2021 W. Alex Ronke, a.k.a. NoPunIn10Did (w.alex.ronke@gmail.com)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+[0] = LAYOUT_lh(
+
+KC_HOME,KC_END ,KC_PGUP,KC_PGDN,KC_ESC ,KC_F1  ,KC_F2  ,KC_F3  ,KC_F4  ,KC_F5  ,KC_F6  ,                KC_F7  ,KC_F8  ,KC_F9  ,KC_F10 ,KC_F11 ,KC_F12 ,KC_PSCR,KC_SLCK,KC_INS ,KC_DEL ,
+KC_NLCK,KC_PSLS,KC_PAST,KC_PMNS,        KC_GRV ,KC_1   ,KC_2   ,KC_3   ,KC_4   ,KC_5   ,                KC_6   ,KC_7   ,KC_8   ,KC_9   ,KC_0   ,KC_MINS,KC_EQL ,KC_BSPC,KC_BSPC,        
```

Comment:
```suggestion
KC_HOME,KC_END ,KC_PGUP,KC_PGDN,KC_ESC ,KC_F1  ,KC_F2  ,KC_F3  ,KC_F4  ,KC_F5  ,KC_F6  ,                KC_F7  ,KC_F8  ,KC_F9  ,KC_F10 ,KC_F11 ,KC_F12 ,KC_PSCR,KC_SCRL,KC_INS ,KC_DEL ,
KC_NUM ,KC_PSLS,KC_PAST,KC_PMNS,        KC_GRV ,KC_1   ,KC_2   ,KC_3   ,KC_4   ,KC_5   ,                KC_6   ,KC_7   ,KC_8   ,KC_9   ,KC_0   ,KC_MINS,KC_EQL ,KC_BSPC,KC_BSPC,        
```

Reply:
Because this code will not compile otherwise.

Reply:
It's already compiling.

Reply:
It won't on develop. These keycodes were deprecated and have been removed:
https://github.com/qmk/qmk_firmware/blob/master/quantum/keycode_legacy.h

Reply:
That's a bad change on develop for some of these. I'm okay with num lock and scroll, but the International keycode constants should be changed back.

Reply:
They won't be. The keycode names as they are now more closely follow what the HID usage tables document calls them. Language-specific aliases are contained in individual headers:

https://docs.qmk.fm/#/reference_keymap_extras

Reply:
Frustrating that the HID spec isn't more descriptive in its own labeling of those keys, particularly since it uses "Non-US # and ~", which is specifically the UK assignment of that key.  International keys 1-3 get extremely limited usage since they are excluded on the vast majority of keyboard hardware.  I'll make the change.